style/core.less

```diff
@@ -4,6 +4,8 @@
   // https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context
   // We set a stacking context so internal elements don't render on top of external components.
   contain: strict;
+  contain: size layout style paint;
+  content-visibility: auto;
```

Comment:
Just so I understand, we will eventually remove the `contain` style once `content-visibility` has a better support. Correct?

Reply:
I think so yes.